src/v/security/audit/schemas/types.h

```diff
@@ -206,6 +206,17 @@ struct authorization_result {
     auto equality_fields() const { return std::tie(decision, policy); }
 };
 
+// A collection or association of entities, such as users, policies, or devices.
+// https://schema.ocsf.io/1.0.0/objects/group?extensions=
+struct group {
+    enum class type_id : int { unknown = 0, role = 1 };
```

Comment:
nit: if `int` is there for serde, it's unnecessary since serde will override and always use a fixed-size integer type. otherwise, int is already the default i think.

Reply:
Fair enough, I was just following the pattern from:
```c++
// Characteristics of a user/person or security principal
// https://schema.ocsf.io/1.0.0/objects/user?extensions=
struct user {
    enum class type : int {
```